src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -181,12 +182,16 @@ class ActivityFeed extends React.Component<Props, State> {
         }
     };
 
-    handleToggleHeaderShadow = () => {
-        if (this.feedContainer) {
-            const shouldHaveShadow = this.feedContainer.scrollTop !== 0;
+    handleFeedScroll = (event: SyntheticEvent<HTMLDivElement>): void => {
+        const { target } = event
+        if (target instanceof Element) {
```

Comment:
When would this be something besides an Element?

Reply:
Probably never. It's there to satisfy flow type checker. Without it type checker returns with error
```
Property scrollTop is missing in EventTarget [1].
```